inc/layers.php

```diff
@@ -817,6 +806,10 @@ function get_layer($post_id = false) {
             //, 'legend' => $layer_legend
         );
 
+        if (!empty($in_category)):
```

Comment:
@Huyeng this solves an error which appears when there is no category on the wordpress instance (as it was the case on my local)